src/test.python.bridge.cpp

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 The Foundry Visionmongers Ltd
+
+#include <memory>
+
+#include <Python.h>
```

Comment:
Would move these to the end of the #include list, and swap their order.

Reply:
Discussed further in https://github.com/OpenAssetIO/OpenAssetIO-Test-CMake/pull/1#discussion_r1013815282